centreon/packages/ui/src/StatusChip/index.tsx

```diff
@@ -1,65 +1,13 @@
-import { ReactNode } from 'react';
+import { ReactNode } from "react";
```

Comment:
I think you have to recalibrate the eslint in your VSCode. You can use this [documentation](https://centreon.atlassian.net/wiki/spaces/RD/pages/1370652724/Integrate+ESLint+to+VSCode)